packages/app/src/specs/SpecsListRowItem.vue

```diff
@@ -1,14 +1,40 @@
 <template>
-  <div
-    class="h-full outline-none border-gray-50 ring-inset grid grid-cols-2 group focus-within:ring-indigo-300 focus-within:ring-1 children:cursor-pointer"
-    data-cy="specs-list-row"
-  >
-    <div>
-      <slot name="file" />
-    </div>
-
-    <div>
-      <slot name="git-info" />
-    </div>
+  <div data-cy="specs-list-row">
+    <component
+      :is="isLeaf ? 'RouterLink' : 'div'"
+      class="h-full outline-none border-gray-50 ring-inset grid group focus:outline-transparent focus-within:ring-indigo-300 focus-within:ring-1 children:cursor-pointer"
+      :class="[isLeaf ? 'grid-cols-2' : 'grid-cols-1']"
```

Comment:
I noticed that the directory rows render a little off-center:

![Screen Shot 2022-06-06 at 4 24 39 PM](https://user-images.githubusercontent.com/1711637/172252447-b03ee63a-a8d7-47d0-8438-9d4c78b29c51.png)

I think that's because we flex this class to `grid-cols-1` when it's not a leaf, but we continue to render the git info div on line 17 regardless.

That kinda has me thinking why we need the git-info slot here at all? Could we instead make this a concern of the `SpecItem` component and consolidate all the 'leaf'-related logic in one place?

Reply:
I'm not sure if this is still the case but we used to have multiple views for this table, and I think some code was shared with the spec list on the runner page. That might explain some slots that don't seem to do much.

It's possible some old code is still hanging around from when we were still figuring the UI for this page out. Don't be afraid to delete code that seems weird/incorrect/not useful.

Reply:
I don't think this is a dead/legacy code issue, rather just that we currently don't display anything in the "Last Updated" slot for directories from the newly-added Git Info data in this table. Since there's a non-zero chance we may decide to put something in that column for directories in the future I reverted the style change so the directory renders as expected and just has an empty second column